org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -22,51 +22,52 @@ <h2>Snapshot Build @qualified.bundle.version@ (@build.date@)</h2>
+      and <code>provided</code> dependencies
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/498">#498</a>,
+      <a href="https://github.com/jacoco/jacoco/issues/572">#572</a>).</li>
+</ul>
+
+<p>
+  During creation of reports various compiler generated artifacts are filtered
+  out, which otherwise require unnecessary and sometimes impossible tricks to
+  not have partial or missed coverage:
+</p>
+
+<ul>
+  <li>Methods <code>valueOf</code> and <code>values</code> of enum types
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/491">#491</a>).</li>
-  <li>Exclude from a report a part of bytecode that compiler generates for a
-      synchronized statement
+  <li>Private empty no-argument constructors
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/529">#529</a>).</li>
+  <li>Methods annotated with <code>@lombok.Generated</code> to better integrate
+      with Lombok >= 1.16.14. Initial analysis and contribution by Rüdiger zu Dohna
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/513">#513</a>).</li>
+  <li>Methods annotated with <code>@groovy.transform.Generated</code> to better
+      integrate with Groovy >= 2.5.0. Thanks to Andres Almiray for adding the annotation to Groovy
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/610">#610</a>).</li>
+  <li>Part of bytecode for <code>synchronized</code> blocks
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/501">#501</a>).</li>
-  <li>Exclude from a report a part of bytecode that compiler generates for a
-      try-with-resources statement
+  <li>Part of bytecode for try-with-resources statements
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/500">#500</a>).</li>
-  <li>During creation of a report merge duplicate instructions that compiler generates
-      for finally blocks
+  <li>Part of bytecode for <code>finally</code> blocks
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/604">#604</a>).</li>
-  <li>Exclude from a report methods which are annotated with <code>@lombok.Generated</code>.
-      Initial analysis and contribution by Rüdiger zu Dohna
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/513">#513</a>).</li>
-  <li>Exclude from a report methods which are annotated with
-      <code>@groovy.transform.Generated</code> to better integrate with Groovy
-       >= 2.5.0. Thanks to Andres Almiray for adding the annotation to Groovy
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/610">#610</a>).</li>
-  <li>Exclude from a report private empty constructors that do not have arguments
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/529">#529</a>).</li>
-  <li>Exclude from a report a part of bytecode that javac generates for a
-      String in switch statement
+  <li>Part of bytecode for <code>switch</code> statements on
+      <code>java.lang.String</code> values
```

Comment:
@marchof #596 is only about `javac`, don't know if it's worth mentioning or not?